README.md

```diff
@@ -23,35 +23,50 @@ The jsep built files will be in the build/ directory.
 
 #### Client-side
 
-```javascript
-<script src="/PATH/TO/jsep.min.js"></script>
-...
-let parse_tree = jsep("1 + 1");
+```html
+<script type="module">
+import { Jsep } from '/PATH/TO/jsep.min.js';
+const parsed = Jsep.parse('1 + 1');
+</script>
 ```
 
 #### Node.JS
 
 First, run `npm install jsep`. Then, in your source file:
 
 ```javascript
-let jsep = require("jsep");
-let parse_tree = jsep("1 + 1");
+// ESM:
+import { Jsep } from 'jsep';
+const parse_tree = Jsep.parse('1 + 1');
+
+// or:
+import jsep from 'jsep';
+const parse_tree = jsep('1 + 1');
+
+
+// CJS:
+const { Jsep } = require('jsep');
+const parse_tree = Jsep.parse('1 + 1');
+
+// or: (NOTE: versions prior to 1.x didn't require `.default`)
+const jsep = require('jsep').default;
```

Comment:
@LeaVerou or @EricSmekens ... not sure if we want to leave it like this? Or commonjs has to require from a different folder to get the named export?

Reply:
Yeah I think it might be better if we could avoid this, but no strong opinion.

Reply:
I was able to fix this ... it's a double build though to avoid combining named and default exports 🤷🏻‍♀️  (for this small a project, seems okay?)